.travis.yml

```diff
@@ -27,6 +27,23 @@ jobs:
           script:
           - yarn test
 
+        - stage: gatsbygram ui tests
+          language: node_js
```

Comment:
@m-allanson Ah, didn't know that ENV variables are not accessible before merging. Nope, I didn't try adding the variable here. Adding it here will make it public right? I'm not sure if it's okay. From [Cypress docs](https://docs.cypress.io/guides/core-concepts/dashboard-service.html#Identification):
>If you have a public project you should still keep your record key secret. If someone knows both your record key and your projectId, they could record test runs for your project - which would mix up all of your results!

Wonder if it's okay for us to make it public? If yes, we can just pass the key from the npm script itself.

Reply:
> Ah, didn't know that ENV variables are not accessible before merging. 

Oh hrmm, it seems I was wrong about that, Travis will use the yaml file from the current branch: https://docs.travis-ci.com/user/customizing-the-build/#Building-Specific-Branches

Yeah adding it here will make it public, but you can use the Travis CLI to [encrypt env vars](https://docs.travis-ci.com/user/environment-variables/#Defining-encrypted-variables-in-.travis.yml). See [an example here](https://github.com/tsriram/gatsby/blob/6b9edbbaa6825d3fc41429fd4fc517426973f15e/.travis.yml#L49-L51). 

Although pay attention to their note:

> Encryption and decryption keys are tied to the repository. If you fork a project and add it to Travis CI, it will not have access to the encrypted variables.

Otherwise you may spend a lot of time (like I did 😅) wondering why your env var isn't working. It may be easier to merge this branch as-is, and then create smaller PRs straight from this repo to tinker with the `travis.yml` config.

Alternatively @KyleAMathews could add the env var via the Travis site (as you mentioned originally) which would certainly be quicker :), but I think it's nicer to keep the Travis config under version control where possible.

Reply:
>Oh hrmm, it seems I was wrong about that, Travis will use the yaml file from the current branch
Probably not :smile: Looks like the environment variables defined in repository settings are not "automatically" available to forks: https://docs.travis-ci.com/user/environment-variables/#Defining-Variables-in-Repository-Settings. So, as you said it's pr